change/@ni-nimble-components-063e914a-8704-466c-b58f-6ad350207e57.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@atmgrifter00 @jattasNI talked to @munteannatan offline and was told that they believe all current issues expressed by Meyers should be covered in the HLD update. Said as y'all may have limited availability this week said to go ahead and move to owners review to get more feedback.